token/common.tact

```diff
@@ -93,3 +93,33 @@ struct BurnNotification {
 
     @derive impl Serialize {}
 }
+
+contract interface JettonOwner {
+    msg receive_notification(prefix: 0x7362d09c, notification: TransferNotification);
+}
+
+contract interface ResponseReceiver {
+    msg receive_response(prefix: 0xd53276db, response: Excesses);
+}
+
+contract interface Master {
+    msg receive_burn_notification(prefix: 0xd53276db, notification: BurnNotification);
+}
+
+contract interface Jetton {
+    var balance: Coins;
```

Comment:
```suggestion
    var balance: JettonAmount;
```

Let's reserve "Coins" for toncoins (aka Grams in prehistoric TON) - they have standard divisibility. Jetton is not a coin, it's an issued token, and the issuer decides its divisibility. Different minters would have different divisibility and limits.

Reply:
Fixed in [cbd3aa4](https://github.com/tact-lang/tact-bikeshed/pull/5/commits/cbd3aa4d496e4166b3db6bb6b09ba90e6d603012)